templates/dashboard/index.html.twig

```diff
@@ -40,10 +40,17 @@
                                                     {% set spf_pass = false %}
                                                   {% endif %}
                                               {% endfor %}
-                                              <span class="btn btn-sm btn-{% if spf_pass %}success{% else %}danger{% endif %}">spf</span>
-                                              <span class="btn btn-sm btn-{% if dkim_pass %}success{% else %}danger{% endif %}">dkim</span>
+                                              <span class="badge badge-pill text-bg-{% if spf_pass %}success{% else %}danger{% endif %}">SPF</span>
+                                              <span class="badge badge-pill text-bg-{% if dkim_pass %}success{% else %}danger{% endif %}">DKIM</span>
+                                            </td>
+                                            <td>
+                                              <a href="{{ path('app_dmarc_reports_report', {report: report.id}) }}" class="link-primary">
+                                                <!-- Bootstrap icon 'eye-fill' -->
```

Comment:
This project does not use the bootstrap icons but the hero-icons (https://heroicons.com/).
this should be consistent throughout the entire project so please find a icon from the hero-icons instead

Reply:
I did not find any reference to icons, all I could find were hardcoded.
Good to know, did you document this anywhere?

Reply:
i did not (yet)